src/components/ChatEntry.js

```diff
@@ -1,22 +1,45 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const source = props.sender === 'Vladimir' ? 'local' : 'remote';
```

Comment:
Good use of the ternary operator! Also good way to distinguish the remote user from the local user!

Further, I think within the hierarchy this is exactly correct place for this check. It wouldn't make sense higher up.